src/app/components/shared/model-cards/card/card.component.ts

```diff
@@ -90,43 +37,43 @@ import { WithLoadingPipe } from "../../../../pipes/with-loading/with-loading.pip
     AsyncPipe,
     WithLoadingPipe,
   ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CardComponent implements OnInit {
+export class CardComponent {
   private readonly recordingApi = inject(AudioRecordingsService);
   private readonly session = inject(BawSessionService);
   private readonly licenseService = inject(LicensesService);
 
-  @Input() public model: Project | Region;
+  public readonly model = input<Project | Region>();
 
-  public hasNoAudio$: Observable<boolean>;
-  protected isOwner: boolean;
-  protected licenseText: string | undefined = undefined;
+  protected readonly license = resource({
```

Comment:
i wonder if we should suffix this. `license` is not a signal or a value... it's a wrapper for fetching the value.

`licenseResource`?

Angular docs do it: https://angular.dev/guide/signals/resource